index.js

```diff
@@ -2,12 +2,7 @@
 module.exports = function (config) {
   var library = config.mock === false ?  // mock is on by default
       require('./trim.production') :
-      require('./trim.development');
+      require('./trim.development')
```

Comment:
FWIW (I know its out of scope of this PR)
I'm not a fan of the production vs development. Best to only support a single code base. If mocks are needed, they should be as needed in the consuming project.